os/binfmt/binfmt_loadbinary.c

```diff
@@ -111,6 +123,15 @@ int load_binary(FAR const char *filename, size_t binsize, size_t offset)
 
 	sched_lock();
 
+#ifdef CONFIG_APP_BINARY_SEPARATION
+	/* The first 4 bytes of the text section of the application must contain a
+	pointer to the application's mm_heap object. Here we will store the mm_heap
+	pointer to the start of the text section */
+	*(uint32_t *)(bin->alloc[0]) = (uint32_t)start_addr;
+	tcb = (struct tcb_s*)sched_self();
```

Comment:
Please add space between 'tcb_s' and '*'

Reply:
Corrected